src/control/cmd/dmg/pool_test.go

```diff
@@ -116,7 +116,31 @@ func TestPoolCommands(t *testing.T) {
 			"Create pool with missing arguments",
 			"pool create",
 			"",
-			errMissingFlag,
+			errors.New("must be supplied"),
+		},
+		{
+			"Create pool with incompatible arguments (auto)",
+			fmt.Sprintf("pool create --size %s --nvme-size %s", testScmSizeStr, testScmSizeStr),
```

Comment:
Should probably have cases for all the bad combinations, right? i.e. A version of this with --scm-size

Reply:
Done.